conversionSetup.sh

```diff
@@ -4,9 +4,8 @@
 # install pandoc
 sudo apt-get install pandoc
 
-# install python3.8
-sudo apt install python3.8
-
 #pip3 install pdfminer
 sudo apt install python3-pdfminer
 
+#install poppler utils
+sudo apt-get install poppler-utils
```

Comment:
what is this package used for?

Reply:
Poppler-utils offers a few options for converting and extracting information from a PDF. I use its function for extracting the text from a PDF in isTextRetained.py to ensure all text is retained.